### PR TITLE
Fix/long tables with bullets

### PR DESCRIPTION
**Pull request by stevejpurves**
* No actual long table support here
* bullet lists that work in tables
* calculating column widths for any table environment

## todo
- [ ] tests for column widths algo

### CODE REVIEW

File: src/nodes/figure.ts [hunk 1]
```diff
@@ -17,6 +18,9 @@ import { writeDirectiveOptions } from '../serialize/markdown/utils';
 
 export type Attrs = NumberedNode & {
   align: AlignOptions;
+  long: boolean;
+  landscape: boolean;
+  fullPage: boolean;
```
Comment (rowanc1) [inline src/nodes/figure.ts:23]: lowercase?!

File: src/nodes/figure.ts [hunk 2]
```diff
@@ -26,14 +30,20 @@ const figure: MyNodeSpec<Attrs> = {
   attrs: {
     ...getNumberedDefaultAttrs(),
     align: { default: 'center' },
+    long: { default: false },
+    landscape: { default: false },
+    fullPage: { default: false },
   },
   toDOM(node) {
-    const { align } = node.attrs;
+    const { align, long, landscape, fullPage } = node.attrs;
```
Comment (stevejpurves) [inline src/nodes/figure.ts:38]: lowercase

File: src/nodes/figure.ts [hunk 3]
```diff
@@ -45,6 +55,9 @@ const figure: MyNodeSpec<Attrs> = {
         return {
           ...getNumberedAttrs(dom),
           align: dom.getAttribute('align') ?? 'center',
+          long: dom.getAttribute('long') ?? false,
+          landscape: dom.getAttribute('landscape') ?? false,
+          fullPage: dom.getAttribute('full-page') ?? false,
```
Comment (stevejpurves) [inline src/nodes/figure.ts:60]: hypenate???

Comment (rowanc1) [inline src/nodes/figure.ts:60]: readBoolDomAttr please!

**Review by rowanc1 — COMMENTED** [review]
naming comments.

**Pull request merged**